components/Household.js

```diff
@@ -0,0 +1,259 @@
+      <H2>{household?.name || "-unbekannter Haushalt-"}</H2>
+      <List>
+        <StyledMenu
+          width={20}
+          height={20}
+          onClick={() => setMenuVisible(!menuVisible)}
+          $rotate={menuVisible}
+        />
+
+        {menuVisible && (
+          <MenuContainer
+            toggleMenu={() => setMenuVisible(!menuVisible)}
+            right="1rem"
+            top="3rem"
+          >
+            {userIsHouseholdAdmin && (
+              <>
+                <UnstyledButton
+                  onClick={() => {
+                    setIsChangeHouseholdName(true);
+                    setIsEditingHousehold(true);
+                    setMenuVisible(!menuVisible);
+                  }}
+                >
+                  <Pen width={15} height={15} />
+                  Haushalt bearbeiten
+                </UnstyledButton>
+                <UnstyledButton
+                  onClick={() => {
+                    setIsAddToHousehold(true);
+                    setMenuVisible(false);
+                  }}
+                >
+                  <Plus width={15} height={15} />
+                  Mitglied hinzufügen
+                </UnstyledButton>
+              </>
+            )}
+            <UnstyledButton
+              onClick={() => {
+                setIsChangeHousehold(!isChangeHousehold);
+                setMenuVisible(false);
+              }}
+            >
+              <Reload width={15} height={15} />
+              Haushalt wechseln
+            </UnstyledButton>
+          </MenuContainer>
+        )}
+        {!isChangeHouseholdName ? (
+          <>
+            {isChangeHousehold && (
+              <ModalComponent toggleModal={() => setIsChangeHousehold(false)}>
+                <Select
+                  value={selectedHouseholdId}
+                  onChange={handleHouseholdChange}
+                >
+                  {user?.households.map((household) => (
+                    <option key={household._id} value={household._id}>
+                      {household.name}
+                    </option>
+                  ))}
+                </Select>
+              </ModalComponent>
+            )}
+
+            {household?.members.find((member) => member._id === user._id)
+              .role !== "owner" && (
+              <button
+                onClick={async () => {
+                  await leaveHousehold(selectedHouseholdId, user._id);
+                  await mutateUser();
+                  notifySuccess("Haushalt verlassen.");
+                }}
+              >
+                Haushalt verlassen
+              </button>
+            )}
+          </>
+        ) : (
+          <form onSubmit={changeHouseholdName}>
+            <input
+              type="text"
+              placeholder="neuer Haushalts-Name"
+              defaultValue={household.name}
+              name="newName"
+              aria-label="neuer Name für den Haushalt"
+              required
+            ></input>
```

Comment:
Inputs should be self closing. 

```suggestion
            <input
              type="text"
              placeholder="neuer Haushalts-Name"
              defaultValue={household.name}
              name="newName"
              aria-label="neuer Name für den Haushalt"
              required
            />
```